### PR TITLE
CB-007: add repositories and domain class to access db

### PR DESCRIPTION
**Pull request by achyutdev**

### CODE REVIEW

File: gradle.properties [hunk 1]
```diff
@@ -1 +1 @@
-appVersion=0.0.1
+appVersion=0.0.2
```
Comment (sbam1) [inline gradle.properties:1]: is this an our app version I think this should be part of gradle build file??

File: src/main/java/org/emela/chautari/domain/RentalEntity.java [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+@Data
+@Entity(name = "rental")
+@NoArgsConstructor
+public class RentalEntity {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private long id;
+
+    @GeneratedValue(strategy = GenerationType.AUTO)
```
Comment (sbam1) [inline src/main/java/org/emela/chautari/domain/RentalEntity.java]: https://stackoverflow.com/questions/45086957/how-to-generate-an-auto-uuid-using-hibernate-on-spring-boot

File: src/main/java/org/emela/chautari/domain/RentalEntity.java [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+    private List<PreferenceEntity> preferences;
+
+    @OneToMany(cascade = CascadeType.ALL, targetEntity = FeatureEntity.class)
+    @JoinColumn(name = "id" )
+    private List<FeatureEntity> features;
+
+
+    private ZonedDateTime postedOn;
+
+    private String resourceIds;
```
Comment (sbam1) [inline src/main/java/org/emela/chautari/domain/RentalEntity.java]: how to get this? do we need mapping?

File: src/main/java/org/emela/chautari/domain/ResourceEntity.java [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+public class ResourceEntity {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private long id;
+
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private UUID resourceId;
+
+    private UUID userId;
```
Comment (sbam1) [inline src/main/java/org/emela/chautari/domain/ResourceEntity.java]: not sure without mapping how to populate these values and correct mapping?

**Review by sbam1 — COMMENTED** [review]
got few questions otherwise looks good, thanks

File: gradle.properties (hunk 1 above)
Comment (achyutdev) [reply gradle.properties:1]: Yes, gradle build uses this file to create artifact version

File: src/main/java/org/emela/chautari/domain/RentalEntity.java (hunk 3 above)
Comment (achyutdev) [reply src/main/java/org/emela/chautari/domain/RentalEntity.java]: Yes we need mapping. I missed it.

**Review by sbam1 — APPROVED** [review]

**Pull request merged**